iac-extensions/docker/src/test/resources/checks/MisusedUnsetCheck/MisusedUnsetCheck.dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM scratch
+
+ENV SECRET_VALUE=password SECRET_USER=admin
+ARG SECRET_TOKEN=0x42f42f42f
+
+# Noncompliant@+1 {{Use the ARG instruction or set and unset the environment variable in a single layer.}}
+RUN ./script -p $SECRET_VALUE && unset SECRET_VALUE
+#                                      ^^^^^^^^^^^^
+# Noncompliant@+1
+RUN ./script -p $SECRET_VALUE && unset -v SECRET_VALUE
+#                                         ^^^^^^^^^^^^
+# Noncompliant@+1 2
+RUN ./script -p $SECRET_VALUE && unset SECRET_VALUE SECRET_USER
+# Noncompliant@+1
+RUN ./script -p $SECRET_VALUE && unset LD_LIBRARY_PATH SECRET_USER
```

Comment:
What happens if we write sudo unset? Will this break the implementation with `.get(0)`

Reply:
No, the `get(0)` refers to `Argument#expressions()`, while `sudo` will be just another of arguments. It won't be captured by `CommandDetector`. But thanks for the remark, I'll add a test case.